app/src/androidTest/java/com/github/se/polyfit/data/remote/firebase/MealFirebaseRepositoryTest.kt

```diff
@@ -96,19 +95,17 @@ class MealFirebaseRepositoryTest {
   }
 
   @Test
-  fun storeMealFailureWithEmptyFirebaseId() = runBlocking {
+  fun storeMealFailureWithEmptyFirebaseId(): Unit = runBlocking {
     every { mockCollection.document(any()) } returns mockDocumentReference
     every { mockDocumentReference.set(any()) } returns
         Tasks.forException(Exception("Failed to store meal"))
     every { mockCollection.add(any()) } returns
         Tasks.forException(Exception("Failed to store meal"))
     every { mockDocumentReference.id } returns "1"
 
-    val exception =
-        assertFailsWith<Exception> {
-          runBlocking { mealFirebaseRepository.storeMeal(mealNoId).await() }
-        }
-
+    assertFailsWith<Exception> {
+      runBlocking { mealFirebaseRepository.storeMeal(mealNoId).await() }
```

Comment:
Can't the runBlocking be removed here as it is already in a runBlocking block ?

```suggestion
       mealFirebaseRepository.storeMeal(mealNoId).await() 
```

Reply:
I'm not sure :p I'll check